tests/test_scripts.py

```diff
@@ -10,9 +17,10 @@
         for name, ep in crmprtd.pkg_resources.get_entry_map("crmprtd")[
             "console_scripts"
         ].items()
+        if name != "crmprtd_download"
```

Comment:
At this point the set of entry points excluding `crmprtd_download` is only of length 2. Maybe simpler to just enumerate them?